styles/globals.css

```diff
@@ -1,3 +1,17 @@
+:root {
```

Comment:
nit:

Let's divide it into 2 files
- `base.css`
- `colors.css` 

It seems that it will be better this way. By definition they will be `globals` if there is no `.module`

Reply:
\+